x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/305_connector_post.yml

```diff
@@ -105,6 +105,16 @@ setup:
           index_name: search-test
 
 
+---
+'Create Connector - Without index attached':
+  - do:
+      connector.post:
+        body:
+          service_type: super-connector
+
+  - set:  { id: id }
+  - match: { id: $id }
```

Comment:
Could we use `exists` here?

Reply:
yeah will add exist check and additionally assert that the index name is null

Reply:
Actually, it's not possibe: https://github.com/elastic/elasticsearch/blob/main/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/README.asciidoc#exists exists assumes that the value is non-null so can't check for `index_name`. I added `match` assertions